DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -157,6 +292,899 @@ def getDateTime(self, uid):
     except ValueError:
       dateTime = ""
     return dateTime
+  
+  def checkUsePlugin(self, srFileName):
```

Comment:
I would rename this function to something more intuitive. For example `containsPlanarAnnotations`